operations/validate.go

```diff
@@ -132,6 +132,14 @@ func validateFile(path string, ac *legacyClient, quiet, includeLong bool, localM
 		return errors.Wrapf(err, "Could not marshal parser project into yaml")
 	}
 
+	if pc != nil {
+		projectConfigYaml, err := yaml.Marshal(pc.ProjectConfigFields)
+		if err != nil {
+			return errors.Wrapf(err, "Could not marshal project config into yaml")
+		}
+		projectYamlString := fmt.Sprintf("%s\n%s", string(projectYaml), string(projectConfigYaml))
```

Comment:
sorry i know i recommended this but I wondder if bytes.Join might be the better way to do this
https://pkg.go.dev/bytes#Join